src/panels/lovelace/cards/hui-entity-filter-card.js

```diff
@@ -72,35 +59,33 @@ class HuiEntitiesCard extends PolymerElement {
     if (this.lastChild) {
       this.removeChild(this.lastChild);
     }
-    let error = null;
-    let cardConfig;
-    let tag = config.card ? computeCardElement(config.card) : 'hui-entities-card';
+    let error;
+    let element;
 
-    if (tag === null) {
-      error = `Unknown card type encountered: "${config.card}".`;
-    } else if (!customElements.get(tag)) {
-      error = `Custom element doesn't exist: "${tag}".`;
-      if (!(tag in this._whenDefined)) {
-        this._whenDefined[tag] = customElements.whenDefined(tag)
-          .then(() => this._configChanged(this.config));
-      }
-    } else if (!config.filter || !Array.isArray(config.filter)) {
-      error = 'No or incorrect filter.';
+    if (!config.filter || !Array.isArray(config.filter)) {
+      error = 'Incorrect filter config.';
+    } else if (!config.card) {
+      config.card = { type: 'entities' };
+    } else if (!config.card.type) {
+      config.card.type = 'entities';
     }
+
+
     if (error) {
-      tag = 'hui-error-card';
-      cardConfig = { error };
+      element = document.createElement('hui-error-card');
```

Comment:
Any reason you wouldn't have the `hui-error-card` be created via `createCardElement`? It's a card

Reply:
What about we do something like this:

```js
const cardConfig = error ? createErrorConfig(error, config.card) : config.card
const element = createCardElement(cardConfig);
```

That way `createCardElement` does the card creation and we have a helper function to generate configs for error cards. That way we can also easily upgrade the error config if we change the card.